enumerables.rb

```diff
@@ -84,8 +123,59 @@ def my_inject(acumulator = nil, operator = nil)
   def multiply_els
     my_inject(:*)
   end
```

Comment:
multiply_els is not an enumerable method to it should be placed outside of the module. It accepts an array as an argument and multiplies all the elements of the array together using #my_inject. A method accepts an array as an argument. So the final outcome of this function should look like...
```ruby
multiply_els(arr)
  arr.my_inject(3, :*)
end
```

Reply:
moved the multiply outside the enumerables